lib/operations/tables.js

```diff
@@ -35,7 +37,7 @@ function parseColumns(columns, table_name, extending_type_shorthands = {}) {
       constraints.push(`CHECK (${options.check})`);
     }
     if (options.references) {
-      constraints.push(`REFERENCES ${options.references}`);
+      constraints.push(`REFERENCES "${schemalize(options.references)}"`);
```

Comment:
I would write it as 
```
template`REFERENCES "${options.references}"`
```

Reply:
Nice, I hadn't noticed that the `template` function used `schemalize`. I've made the requested change.